lib/rummager.rb

```diff
@@ -30,6 +30,7 @@
 require 'unf'
 require 'uri'
 require 'yaml'
+require 'zlib'
```

Comment:
It'd be better to move this into the other file, I think. Keep the require where it's used.

Reply:
we previously moved all requires that are used in the app to this file for clarity - basically, it helped identify strange includes in the project. and doesn't affect load time as they are required on load (in production). 

I personally prefer this approach.